packages/nextjs/pages/api/e2e-tests/mocks.ts

```diff
@@ -0,0 +1,23 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (process.env.NEXT_PUBLIC_API_MOCKING !== "enabled") {
+    res.status(404).send("Error: mocks not enabled");
+    return;
+  }
+
+  const { getMockData, setMockData } = await import("mocks/msw/db/mock-data");
+
+  if (req.method === "GET") {
+    const mockData = getMockData();
```

Comment:
😄 Ah - ok - this answers my question about the randomness and not knowing what values you can check for.

However, I still have a little red light flashing in my head that if a test should ever fail one time due to certain data, it might be hard to debug it since the data set that caused the issue may be hard to reproduce?

Reply:
That's a great point. I just pushed a fix! By calling `faker.seed(0)`, we will still get random data, but at least it will be somewhat reproducible between test runs.

Reply:
And, quite by coincidence, that commit proved its worth!  The tests are now failing with: 
```
TestingLibraryElementError: Timed out retrying after 4000ms: Found multiple elements with the text: White Croissant
``` 
But more importantly -- they're failing consistently on that same error!  I ran twice, and it failed both times identically.
So this revealed a problem with my random data ... sometimes it creates duplicate category names.